src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -457,6 +457,18 @@ public class OpenFileCommand : RecordableCommand
         {
             #region Public Class Methods
 
+            /// <summary>
+            /// Constructor
+            /// </summary>
+            /// <param name="filePath">The path to the file.</param>
+            /// <param name="forceManualExecutionMode">Should the file be opened in manual execution mode?</param>
+            public OpenFileCommand(string filePath, bool forceManualExecutionMode = false)
```

Comment:
do you want to obsolete this one?

Reply:
Not really since the old one is still pretty useful..

Reply:
The old API was public `OpenFileCommand(string filePath, bool forceManualExecutionMode = false)`
All 3dParty calls to `OpenFileCommand(filepath)` would now cause an ambiguous exception;

Reply:
@pinzart90 have you verified that? I recall that the overload is actually resolved at compile time - but it's been a long time.

Reply:
so this... it's subtle.
https://stackoverflow.com/a/40628542

Reply:
@pinzart90 I think that's exactly why I need to fix all the other cases in this PR, I could make the other constructor `OpenFileCommand(string filePath, bool isTemplate, bool forceManualExecutionMode = false)` so the original one is not ambiguous, what do you think?

Reply:
It may be ok. We should test with pre compiled binaries  (like from revit) that call OpenFileCommand("", false); and OpenFileCommand("") running against these changes.

Reply:
one easy way to do that @pinzart90 @QilongTang is to write a test that is compiled against 2.x and run it against the new dynamo core binaries.

Reply:
O yeah, nice. We've done that before

Reply:
I have modified the new constructor to be a strong signature so we do not have API breaking problem as  `public OpenFileCommand(string filePath, bool forceManualExecutionMode, bool isTemplate)`. Notice that I do not mean to obsolete the old constructor because I dont think the callers need to specify the isTemplate all the time, on the contrary, I think the original constructor should still be most useful.